.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## Nature of issue?
+
+- [ ] Found a bug
```

Comment:
@joeyklee This is something of a silly thing (and I know it's pretty standard with other projects) but I find that the checkbox is sometimes confusing due to the way GitHub renders it as `1/5 tasks completed`. I don't necessarily have a better idea but I think that the template could maybe just include the list without the checkboxes and submitters would then delete the ones that don't apply? (I realize this is really minutiae! 😅 )

Reply:
Oh! That's a good point. Let's remove them and have people delete the ones that don't apply as youv'e suggested. I'll make that change. Thanks!